serverless.yml

```diff
@@ -132,6 +143,110 @@ functions:
               alarmName: ["aqts-capture-error-handler-${self:provider.stage}-error-alarm"]
     vpc: ${self:custom.vpc}
 
+  restoreDbCluster:
+    handler: src.handler.restore_db_cluster
+    role: arn:aws:iam::#{AWS::AccountId}:role/csr-Lambda-Role
+    environment:
+      AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      STAGE: ${self:provider.stage}
+      LOG_LEVEL: INFO
+      MAX_RETRIES: 6
```

Comment:
Does anything actually use the `MAX_RETRIES` environment variable?

Reply:
No, I think I've just been blindly copy and pasting it everywhere ... will remove them prior to check-in.